be/src/Project/Services/DownloadCenter/DownloadCenter.Api/Services/DownloadCenter/DownloadCenterService.cs

```diff
@@ -124,11 +124,11 @@ public async Task<PagedResults<IEnumerable<DownloadCenterItemServiceModel>>> Get
 
                 foreach(var categoryId in downloadCenterFileGroup.OrEmptyIfNull().Select(x => x.CategoryId))
                 {
-                    var downloadCenterFileCategory = this.context.DownloadCenterCategoryTranslations.FirstOrDefault(x => x.CategoryId == categoryId && x.Language == model.Language && x.IsActive);
+                    var downloadCenterFileCategory = _context.DownloadCenterCategoryTranslations.FirstOrDefault(x => x.CategoryId == categoryId && x.Language == model.Language && x.IsActive);
```

Comment:
**suggestion:** doing .FirstOrDefault within foreach results in many database calls. Let's retrieve all the needed one before and use .ToList() (like in other examples and PRs that I created)